src/core/thread/mlr_manager.cpp

```diff
@@ -756,19 +758,19 @@ void MlrManager::LogMlrResponse(Error               aResult,
     OT_UNUSED_VARIABLE(aFailedAddresses);
     OT_UNUSED_VARIABLE(aFailedAddressNum);
 
-#if OPENTHREAD_CONFIG_LOG_BBR
+#if (OPENTHREAD_CONFIG_LOG_LEVEL >= OT_LOG_LEVEL_WARN) && (OPENTHREAD_CONFIG_LOG_MLR == 1)
```

Comment:
Do we need `(OPENTHREAD_CONFIG_LOG_LEVEL >= OT_LOG_LEVEL_WARN)` here? I think it's included in `otLogWarnXXX`.

Reply:
not necessary to build the `if...else...` block at all if the log level is smaller than `OT_LOG_LEVEL_WARN`

Reply:
I prefer the current code of this PR because wrapping all calling to `LogMlrResponse` is tedious.